test/loopback.test.js

```diff
@@ -99,9 +100,72 @@ describe('loopback', function() {
         'version',
       ];
 
+      var EXPECTED_GTNODE4 = [
```

Comment:
This makes it very difficult to understand what's the difference between `EXPECTED_GTNODE4` and `EXPECTED`. Please rework the definition of these two arrays in such way, that one contains the shared items and the second adds/removes only what's different.

I find it also very surprising that this test was passing before you started fixing eslint stuff. I want you to investigate this more and find out what is causing the different set of `loopback` properties. I suspect this change may have been caused by enabling "strict mode".